docs/rules/attribute-names.md

```diff
@@ -33,6 +36,55 @@ The following patterns are not warnings:
 @property({attribute: 'camel-case-name'})
 camelCaseName: string;
 
+@property({attribute: 'camel-case-other-name'})
+camelCaseName: string;
+
+@property()
+lower: string;
+```
+
+## Options
+
+You can specify `style` to one of these values `none`, `snake`, `dash` to
+enforce that attribute name is the styled version of property, or `false`.
+
+For example for a property named `camelCaseProp`, expected attribute names are:
+
+| Style | Attribute       |
+|-------|-----------------|
+| none  | camelcaseprop   |
+| dash  | camel-case-prop |
+| snake | camel_case_prop |
+
+The following patterns are considered warnings with `{"style": "dash"}`
+specified:
+
+```ts
+// Using decorators:
+
+@property() camelCaseName: string;
+
+@property({attribute: 'camel-case-other-name'})
```

Comment:
this would be a warning with `style: "dash"`? this is following the style isn't it?

Reply:
Oh, that was the idea of "strict" naming. Maybe it's more than juste style.
I want to force attribute name to be the exact same name as property with only style transformation.
With kebab style and following code
```ts
@poperty()
fooBar: string;
```
I want attribute to be either `foo-bar` or `false`, no other possible value

Reply:
no worries that makes sense 👍 

think we're fine since the existing rule already enforces the names are equal anyway, so we're not changing behaviour really other than introducing other cases